### PR TITLE
Add frontend basic structure

### LINKED ISSUE #29: Set up frontend basic structure
Set up React Routes and folder structure of React components.

### PR DESCRIPTION
**Pull request by hzxcaryn**
Closes #29 

## What's in this PR
- Basic folder structure with dummy pages for the entry points of customer and merchant pages
- Adding of React router
- Adding of deps that will be used for the frontend (muicss , feather icons, styled components)
- Enabling of absolute import, so we don't have to do relative imports like `../../../components` that gets messy over time

## How to test
Fetch this branch and cd into the `web` folder. Run `yarn` to install the dependencies.
Run `yarn start`. 
**Expected output:**
- At `http://localhost:3000/` you should see the words "Customer Explore Page"
- At `http://localhost:3000/merchant` you should see the words "Merchant Landing Page"

### CODE REVIEW

File: web/.eslintignore [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
-src/react-app-env.d.ts
+# build artifacts
+build/*
+coverage/*
+
+# data definition files
 **/*.d.ts
+
+# 3rd party libs
+/src/public/
```
Comment (ifhuang) [inline web/.eslintignore:9]: remove the start / to be consistent with others?

File: web/src/App.tsx [hunk 2]
```diff
@@ -15,28 +15,13 @@
  */
 
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import {BrowserRouter as Router} from 'react-router-dom';
+import Routes from './Routes';
```
Comment (ifhuang) [inline web/src/App.tsx:19]: sort imports?

File: web/src/Routes.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+// Customer Pages
+const CustomerExplorePage = lazy(() => import('components/customer/explore'));
+// Merchant Pages
+const MerchantLandingPage = lazy(() => import('components/merchant/landing'));
+
+const Routes: React.FC = () => (
+  <Suspense fallback={<Loading />}>
+    <Switch>
+      <Route exact path="/" component={CustomerExplorePage} />
```
Comment (ifhuang) [inline web/src/Routes.tsx:13]: why not path="/customer"?

Comment (hzxcaryn) [reply web/src/Routes.tsx:13]: @ifhuang Hmm then wdyt "/" should be? I was thinking like how most listing sites have customer facing pages at "/" and business facing pages under "/businesses" or something, that's why I decided to go with this split.

File: web/src/App.tsx (hunk 2 above)
Comment (hzxcaryn) [reply web/src/App.tsx:19]: @ifhuang How do you want them sorted? I can add linter rules

Comment (hzxcaryn) [reply web/src/App.tsx:19]: Right now I'm doing Builtins -> 3rd party deps -> project-level imports, should I just add a newline between each section?

Comment (hzxcaryn) [reply web/src/App.tsx:19]: Actually I think I'll just deal with all imports ordering in another PR 😃

Comment (ifhuang) [reply web/src/App.tsx:19]: sounds good, and inside each section sort alphabetically

File: web/src/Routes.tsx (hunk 3 above)
Comment (ifhuang) [reply web/src/Routes.tsx:29]: ic, make sense

**Pull request merged**